daemon/daemon_windows.go

```diff
@@ -37,6 +38,8 @@ const (
 	windowsMinCPUCount   = 1
 )
 
+var platformExecRoot = filepath.Join(os.Getenv("programdata"), "docker", "plugins")
```

Comment:
This should be following the config.Root to allow two daemons to run side-by-side with out clashing directories, or when -g is specified on the daemon startup.